### ISSUE TITLE
Scale device image properly

### ISSUE THREAD

**Issue by niteshkumar2000** [FreakyOS/website#5]
* We have defined the size of the device card which contains the device image as 15 rem.
* But there is no specific limitations made for the device image size.
* So it takes up the actual size of the image and fills the entire screen.

What should be done:
* Every card should be in same size across the all devices, irrespective of the size of the image.
* This can be achieved by doing two things
    * By limiting the size of the device image
    * By removing the existing size limitation on device card and finding a work around to scale the entire card properly.

**Comment by niteshkumar2000** [comment]
@jeivardan I think we need to remove the size limitation on device card since we hard coded it, It clearly violated our rule! (Hard coding the values in inline css) 
Need to find a work around for that.

**Comment by jeivardan** [comment]
> @jeivardan I think we need to remove the size limitation on device card since we hard coded it, It clearly violated our rule! (Hard coding the values in inline css)
> Need to find a work around for that.

@niteshkumar2000 Yes! Initially I too thought hard coding the size looks ugly and need to find a way for that so that the device card and images scales properly

**Comment by niteshkumar2000** [comment]
> > @jeivardan I think we need to remove the size limitation on device card since we hard coded it, It clearly violated our rule! (Hard coding the values in inline css)
> > Need to find a work around for that.
> 
> @niteshkumar2000 Yes! Initially I too thought hard coding the size looks ugly and need to find a way for that so that the device card and images scales properly
> 

Yeah let's make this as priority 1 and fix this asap!

**Comment by niteshkumar2000** [comment]
@jeivardan I removed the hard-coded values, and rendered every single card with in column so that they can scale dynamically.

And the for dynamic scaling across devices, I used this as reference, please check this out
https://stackoverflow.com/questions/4684304/how-can-i-resize-an-image-dynamically-with-css-as-the-browser-width-height-chang

**Comment by niteshkumar2000** [comment]
![Screenshot_20200723-182333698](https://user-images.githubusercontent.com/36584997/88306605-91662d80-cd28-11ea-8fd1-aac95f783073.jpg)
Now it looks like this on mobile :D

**Comment by jeivardan** [comment]
> @jeivardan I removed the hard-coded values, and rendered every single card with in column so that they can scale dynamically.
> 
> And the for dynamic scaling across devices, I used this as reference, please check this out
> https://stackoverflow.com/questions/4684304/how-can-i-resize-an-image-dynamically-with-css-as-the-browser-width-height-chang

I'll look into it

**Comment by niteshkumar2000** [comment]
Even though the new revamped site somehow makes it looks good but still i feel we have to limit the size of the image for better look :)

Commit :- 61a5e454743e581dfdad200802a097b5a45513da
